stages/org.osbuild.grub2

```diff
@@ -152,6 +152,11 @@ SCHEMA = """
          "examples": ["fedora"],
          "pattern": "^(.+)$"
       },
+      "efi_root": {
+        "type": "string",
+         "description": "The location of the EFI/ directory",
+         "default": "/boot/efi/EFI"
+      },
```

Comment:
I think the description¹ should clarify that this is a **source** directory for EFI files and also that it only has an effect if `install = True`.


¹ And maybe even the option name?  Though I think the current one is fine.

Reply:
Thank you, I updated the description/code now and also called it `efi_src_dir` to make it clearer from this as well.